pkg/diagnostics/network/objects.go

```diff
@@ -114,9 +112,26 @@ func GetTestPod(podName, nodeName string) *kapi.Pod {
 			},
 		},
 	}
+
+	var trimmedPodImage string
+	imageTokens := strings.Split(testPodImage, "/")
+	n := len(imageTokens)
+	if n < 2 {
+		trimmedPodImage = testPodImage
+	} else {
+		trimmedPodImage = imageTokens[n-2] + "/" + imageTokens[n-1]
+	}
+	if trimmedPodImage == util.NetworkDiagDefaultTestPodImage {
```

Comment:
Requirement for our network diagnostic test pod is to serve/listen for requests using tcp/udp on some port. We don't care if the test pod is running `socat` or simple http web server. We chose `socat` because `openshift/origin-deployer` doesn't serve by default and we picked this image for test pod because it already exists in redhat registry and also available on all the nodes. 
For non default test pod passed by user we expect it be serving by default.